src/Uno.UI.Runtime.Skia.Wpf/Extensions/UI/Xaml/Controls/TextBoxViewExtension.cs

```diff
@@ -38,7 +38,11 @@ public void StartEntry()
 			_contentElement = textBox.ContentElement;
 
 			EnsureWidgetForAcceptsReturn();
-			textInputLayer.Children.Add(_currentInputWidget!);
+
+			if (textInputLayer.Children.Count == 0)
+			{
+				textInputLayer.Children.Add(_currentInputWidget!);
+			}
```

Comment:
I don't understand this change. How is it related to resetting the caret? What behavioral change does it introduce, if any?